blockcypher/src/adapter.ts

```diff
@@ -6,51 +6,52 @@ export type JobSpecRequest = {
   id: string
   data: Record<string, unknown>
 }
-type Callback = (statusCode: number, data: Record<string, unknown>) => void
+type JobSpecResponse = { statusCode: number; data: Record<string, unknown> }
```

Comment:
Looks like we're using this across multiple packages, would be nice if we can store it in one place